src/documents/TextToSpeechAPI.md

```diff
@@ -471,9 +478,16 @@ fluid.textToSpeech({
             <td>
 <pre>
 <code>
+
+// Assuming an existing textToSpeech component
+var availableVoices = textToSpeech.getVoices();
+var voiceToUse = availableVoices[0];
```

Comment:
This part of the example seems strange as you are required to have a pre-existing textToSpeech component in order to configure another component. It may be better to just use the web speech api directly to access the voices and set the `voiceToUse` variable.